app/views/ReactionsView/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { useEffect } from 'react';
+import {
+	View, Text, TouchableOpacity
+} from 'react-native';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { withSafeAreaInsets } from 'react-native-safe-area-context';
+import ScrollableTabView, { ScrollableTabBar } from 'react-native-scrollable-tab-view';
+
+import I18n from '../../i18n';
+import { withTheme } from '../../theme';
+import { themes } from '../../constants/colors';
+import { withDimensions } from '../../dimensions';
+import sharedStyles from './styles';
+import SafeAreaView from '../../containers/SafeAreaView';
+import Emoji from '../../containers/message/Emoji';
+
+import TabPage from './renderPage';
+
+const scrollProps = {
```

Comment:
We're not reusing, so we could add to the scroll directly

Reply:
Removed.